templateflow/tests/test_api.py

```diff
@@ -24,20 +24,18 @@
 \tpages = {313--327},
 \tauthor = {Vladimir Fonov and Alan C. Evans and Kelly Botteron and C. Robert \
 Almli and Robert C. McKinstry and D. Louis Collins},
-\ttitle = {Unbiased average age-appropriate atlases for pediatric studies},
-\tjournal = {{NeuroImage}}
```

Comment:
I'm a little nervous that doi.org are changing their responses. They used to generate Lastname_year identifiers, and now these seem to be only year.

We probably can have a regexp to override whatever ID they generate to the template name.

However, I'm less happy with the references not having a journal entry. Before we go ahead we need to understand why these are changing.

Reply:
Yes, looking more into it, it does seem like a bug. Looks like crossref should have a fix out today, dunno if doi.org will follow suit but we could always switch over to crossref since they have an API as well.

https://gitlab.com/crossref/issues/-/issues/1540

Reply:
doi.org just redirects POST requests over to crossref